keepercommander/commands/audit_alerts.py

```diff
@@ -843,6 +855,138 @@ def execute(self, params, **kwargs):
         AuditSettingMixin.invalidate_alerts()
         logging.info('Alert counts reset to zero')
 
+class AuditAlertEnable(EnterpriseCommand, AuditSettingMixin):
+    def get_parser(self):
+        return alert_enable_parser
+
+    def execute(self, params, **kwargs):
+        enable_all = kwargs.get('enable_all', False)
+        target = kwargs.get('target')
+        
+        if enable_all:
+            if target:
+                raise CommandError('Cannot specify both alert target and --all flag')
+            self._enable_all_alerts(params)
+        else:
+            if not target:
+                raise CommandError('Alert target is required unless using --all flag')
+            self._enable_single_alert(params, target, kwargs)
+    
+    def _enable_single_alert(self, params, target, kwargs):
+        alert = AuditSettingMixin.get_alert_configuration(params, target)
+
+        rq = {
+            'command': 'put_enterprise_setting',
+            'type': 'AuditAlertContext',
+            'settings': {
+                'id': alert.get('id'),
+                'disabled': False
+            }
+        }
+        api.communicate(params, rq)
+        AuditSettingMixin.invalidate_alerts()
+        
+        alert_name = alert.get('name') or f"Alert ID {alert.get('id')}"
+        logging.info(f'Alert "{alert_name}" has been enabled')
+        command = AuditAlertView()
+        command.execute(params, target=target)
+    
+    def _enable_all_alerts(self, params):
+        alerts = self.load_settings(params)
+        if not isinstance(alerts, dict):
+            logging.info('No alerts found')
+            return
+        alert_filter = alerts.get('AuditAlertFilter')
+        if not isinstance(alert_filter, list):
+            logging.info('No alerts found')
+            return
+        
+        enabled_count = 0
+        for alert in alert_filter:
+            alert_id = alert.get('id')
+            if alert_id:
+                rq = {
+                    'command': 'put_enterprise_setting',
+                    'type': 'AuditAlertContext',
+                    'settings': {
+                        'id': alert_id,
+                        'disabled': False
+                    }
+                }
+                api.communicate(params, rq)
+                enabled_count += 1
```

Comment:
```suggestion
        requests = []
        for alert in alert_filter:
            alert_id = alert.get('id')
            if alert_id:
                rq = {
                    'command': 'put_enterprise_setting',
                    'type': 'AuditAlertContext',
                    'settings': {
                        'id': alert_id,
                        'disabled': False
                    }
                }
                requests.append(rq)
        
        if not requests:
            logging.info('No valid alerts found to enable')
            return
        
        logging.info(f'Enabling {len(requests)} alert(s) in parallel...')
        results = api.execute_batch(params, requests)
        
        enabled_count = sum(1 for result in results if result.get('result') == 'success')
```

Hey maybe you can do it this way, you will be able to parallelize some requests as `exectute_batch` option is already prebuilt into commander